test/ExtendedXmlSerializerTest/TestObject/TestClassCollection.cs

```diff
@@ -0,0 +1,28 @@
+﻿// MIT License
+// 
+// Copyright (c) 2016 Wojciech Nagórski
```

Comment:
I prefere adding comments only if it is really needed. But other projects always have copyrights in header of file (eg. Autofac).

I would like to add wiki pages or create page on github, but my English isn't perfect and it takes me a lot of time. But I'll create it ;)

I have only article on Code project: https://www.codeproject.com/articles/1121574/extended-xml-serializer-for-net
and readme and few answers on stackoverflow.

I'll add information about your effort in readme and in article at codeproject.